update/automated.go

```diff
@@ -10,17 +10,17 @@ import (
 )
 
 type Automated struct {
-	changes []change
+	Changes []Change
 }
 
-type change struct {
-	service   flux.ServiceID
-	container cluster.Container
-	imageID   flux.ImageID
+type Change struct {
+	ServiceID flux.ServiceID
+	Container cluster.Container
+	ImageID   flux.ImageID
```

Comment:
Do these types need to be exported for (un)marshaling purposes? I see no new usage outside of this package.